tests/backends/test_dbscript.py

```diff
@@ -591,8 +608,9 @@ def test_inconsistent_params_db_script(fileutils):
             script_path=torch_script,
             device="CPU",
             devices_per_node=2,
+            first_device=0,
```

Comment:
Could we add one more check to this test? Basically just check to make sure that an `SSUnusportedError` is raised when `device="CPU"` & `first_device` is anything other than 0 like like we do when `deivce="CPU"` & `devices_per_node` is anything other than 1.
```py
with pytest.raises(SSUnsupportedError) as ex:
    db_script = DBScript(
         name="test_script_db",
        script_path=torch_script,
        device="CPU",
        devices_per_node=1,
        first_device=123,
    )
assert ex.value.args[0] == "Optional: Check for some msg here"